.github/workflows/UI-Test.yml

```diff
@@ -0,0 +1,26 @@
+name: UI-Test
+
+on:
+  pull_request:
+    branches:
+      - 'master'
+  push:
+    branches:
+      - 'master'
+
+jobs:
+
+  UItest:
+    runs-on: macos-latest
```

Comment:
Why do you need to run this on macos (just curious)? Thanks

Reply:
General note: there's not many comments in this file and it's missing a license header